### DISCUSSION TITLE
Enabled PayPal business and get an error

### DISCUSSION THREAD

**Discussion by naberna** [swellstores/community#304] [Help] [answered]
I enabled paypal business in sandbox and I cannot redirect any page now in admin. Here is the error I get
![image](https://github.com/swellstores/community/assets/73580153/75a8142a-f0f2-4ee5-9b1d-1319e773181b)

**Comment by logeshswell** [comment]
@naberna We identified this bug a few hours back and I will keep you posted if it is resolved.

**Comment by naberna** [comment]
Hello, Thank you very much. I see it is not resolved yet, I need to work on it?

**Reply by logeshswell** [reply]
@naberna Can you help me with your store-id please?

**Reply by naberna** [reply]
@logeshswell I don't know the I'd because I can login but I cannot view any page. The name of the shop is 'Just Check'. I wanted to integrate PayPal in sandbox

**Reply by logeshswell** [reply, marked as answer]
@naberna Your login & redirect issue is fixed now. Can you check and confirm?
If the redirect issue still exists, please share the screen recording of it to debug it faster.

**Reply by naberna** [reply]
@logeshswell thanks yes it is solved. To connect PayPal to my shop, I only need my business account connect with the shop right? Also what how can I add apply and Amazon pay?

**Reply by logeshswell** [reply]
Yes, you're right! Please follow the instructions for setting Paypal and AmazonPay
https://www.swell.is/help/store-setup/alternative-payment-methods/paypal#paypal
https://www.swell.is/help/store-setup/alternative-payment-methods/amazon-pay#amazon-pay
